README.md

```diff
@@ -472,8 +477,60 @@ Create `.eslintrc` at root of your project that contains:
 ```json
 {
   "root": true,
-  "extends": [
-    "@jupiterone/eslint-config/react"
-  ]
+  "extends": ["@jupiterone/eslint-config/react"]
 }
 ```
+
+## Usage: Husky 7
+
+In the updates to Husky for major version 7 they have included some large
+changes that fix existing Husky problems. (Most of these problems revolved
+around failing to properly bootstrap a new cloned project with `.git/hooks`
+entries and the inability to run hooks without first loading the `node.js`
+runtime [due to looking in many different places for a JS config that must be
+added to the `.git/hooks` directory])
+
+A writeup by the Husky authors
+[can be read here](https://blog.typicode.com/husky-git-hooks-javascript-config/).
+Summarized major changes for our use-case are as follows:
+
+- No more `.huskyrc` and no more `[package.json].husky` entries. (Husky is
+  configuring Git with shell scripts now, we don't want JS configs)
+
+- Hooks now live in a directory specified by `git config` at the
+  `core.hooksPath` directory
+
+  - Husky will try to set the above path automatically on `prepare`
+
+- Maintenance needed to upgrade existing hooks to the new format:
+
+### Installation / Upgrade for Previous Projects
```

Comment:
Thanks for adding this section!

Reply:
Ahh, I see this now. I suppose the CHANGELOG just needs a link to this. I'll add it!